<launch>

	<arg name="baseFrequence"		default="100"/>
	<arg name="print"			default="false"/>
	<arg name="NODE_NAME"			default="sound_tracking"/>
	
	<arg name="input_sounds"		default="/sounds/tracked_sources"/>
	<arg name="input_odom" 		    	default="odometry/local_filtered" />
	<arg name="output_cmd_vel"      	default="cmd_vel" />
	
	<!--include file="$(find odas_ros)/launch/odas.launch"/-->

	<node pkg="sound_tracking" name="sound_tracking" type="sound_tracking_node" output="screen">
		<param name="baseFrequence"	type="int"	value="$(arg baseFrequence)"/>
		<param name="print"	type="bool"	value="$(arg print)"/>
		<param name="NODE_NAME" type="string"	value="$(arg NODE_NAME)"/>
		
		<rosparam  	
			command="load" 	
			file="$(find sound_tracking)/config/base_control.yaml" 	
			ns="base_control" />

		<rosparam  	
			command="load" 	
			file="$(find sound_tracking)/config/spatial_filter.yaml" 	
			ns="spatial_filter" />

		<remap from="odas"	to="$(arg input_sounds)" />
		<remap from="cmd_vel"	to="$(arg output_cmd_vel)" />
		<remap from="odom"	to="$(arg input_odom)" />
	</node>

</launch>
